README.md

```diff
@@ -203,14 +212,25 @@ async getEvents(): Promise<Event[]> {
 }
 ```
 
+Fetch a single resource.
+
+```typescript
+async getEvent(id: string, partitionKey: string | string[]): Promise<Event> {
+  const { resource } = await this.eventContainer
+        .item(id, type)
```

Comment:
Hey @manekinekko, What does the type indicate in this context? Perhaps, I might be overlooking something, but the second argument for the "item" is the PartitionKey.

Reply:
The internal type supported by `@CosmosPartitionKey` annotation is:

https://github.com/nestjs/azure-database/blob/9b655157d428ba59f97e8d4d0ee99c3f0e94b1dc/lib/cosmos-db/cosmos-db.decorators.ts#L8-L13

The `type` argument represent the partition key and is a list of strings, eg: `["/name", "/address/zip"]` (as described in [the official sample](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/cosmosdb/cosmos#crud-on-container-with-hierarchical-partition-key))